manticore/platforms/linux.py

```diff
@@ -264,12 +264,21 @@ class Linux(Platform):
     This class emulates the most common Linux system calls
     '''
 
-    def __init__(self, program, argv=None, envp=None):
-        '''
+    def __init__(self, program, argv=None, envp=None, **options):
```

Comment:
My concern was changing the model API whenever we need to add options. Or do we assume all params after `envp` are options?